smack-extensions/src/main/java/org/jivesoftware/smackx/usertune/UserTune.java

```diff
@@ -0,0 +1,82 @@
+/**
+ *
+ * Copyright 2019 Aditya Borikar.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.usertune;
+
+/**
+ * Child elements described as in XEP-0118: User Tune.
+ * @see <a href="https://xmpp.org/extensions/xep-0118.html#table-1"> </a>
+ *
+ * TODO: If the user is listening to a stream, length should not be included.
+ */
+public class UserTune {
+
+    String artist;
```

Comment:
Members of `ExtensionElements` should be private and final.

For complex elements with many members like this, where some members are optional, while others are not, it is advisable to create a `Builder` to construct them. The Builder can be used to require non-optional members.

Reply:
Ah wait, this is not an Element :D Its too early for me :)

Reply:
Created a builder to build UserTuneElement along with static method to create a builder from inside UserTuneElement.